pkg/apis/logging/v1alpha1/register.go

```diff
@@ -0,0 +1,49 @@
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package v1alpha1 contains API Schema definitions for the logging v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=logging.openshift.io
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	monitoring "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+)
+
+var (
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "logging.openshift.io", Version: "v1alpha1"}
+
+	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
+	// SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
```

Comment:
@josefkarasek Any reason why you're using the apimachinery `runtime.NewSchemeBuilder` vs the controller-runtime's wrapper `&scheme.Builder{GroupVersion: SchemeGroupVersion}`?

I'm curious because your types.go and register.go files are quite different from the default that the sdk scaffolds.

As an example if you look at the following:
https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/pkg/apis/cache/v1alpha1/memcached_types.go#L41-L43
https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/pkg/apis/cache/v1alpha1/register.go#L18

For registering custom types not defined under your `pkg/apis/...` I would suggest to register those types with the manager's scheme in your `cmd/manager/main.go`.
https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#register-with-the-managers-scheme

```Go
import (
	monitoring "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
)

func main() {
    ....
    if err := monitoring.AddToScheme(mgr.GetScheme()); err != nil {
      ...
    }
    ....
}
```
That should register all types with the manager's scheme.